tools/update-prod-static

```diff
@@ -33,35 +34,45 @@ os.chdir(settings.DEPLOY_ROOT)
 setup_node_modules(production=True)
 
 # Build emoji
-run(['./tools/setup/emoji/build_emoji'])
+run(["./tools/setup/emoji/build_emoji"])
 
 # Inline CSS in emails
-run(['./scripts/setup/inline_email_css.py'])
+run(["./scripts/setup/inline_email_css.py"])
 
 # Copy over static files from the zulip_bots package
-run(['./tools/setup/generate_zulip_bots_static_files.py'])
+run(["./tools/setup/generate_zulip_bots_static_files.py"])
 
 # Build pygment data
-run(['./tools/setup/build_pygments_data'])
+run(["./tools/setup/build_pygments_data"])
 
 # Create webpack bundle
-run(['./tools/webpack', '--quiet'])
+run(["./tools/webpack", "--quiet"])
 
 # Collect the files that we're going to serve; this creates prod-static/serve.
-run([
-    './manage.py', 'collectstatic', '--no-default-ignore',
-    '-v0',
-    '--noinput',
-    '-i', 'assets',
-    '-i', 'emoji-styles',
-    '-i', 'html',
-    '-i', 'js',
-    '-i', 'styles',
-    '-i', 'templates',
-])
+run(
+    [
+        "./manage.py",
+        "collectstatic",
+        "--no-default-ignore",
+        "-v0",
+        "--noinput",
+        "-i",
+        "assets",
+        "-i",
+        "emoji-styles",
+        "-i",
+        "html",
+        "-i",
+        "js",
+        "-i",
+        "styles",
+        "-i",
+        "templates",
+    ],
```

Comment:
This is kinda a disaster and probably deserves a workaround, perhaps an `exclude_directories` list that gets spliced out.